### PR TITLE
fix number in UnbondRequested and DistributeRewards incorrect

### PR DESCRIPTION
**Pull request by Jianzhuo**

### CODE REVIEW

**Review by ianhe8x — APPROVED** [review]
lgtm

**Comment by mzxyz** [comment]
@Jianzhuo  Please fix the test

File: hardhat.config.ts [hunk 1]
```diff
@@ -55,27 +56,27 @@ task('publish', "verify and publish contracts on etherscan")
 
     try{
         //InflationController
-        await hre.run("verify:verify", {
-            address: deployment.InflationController.innerAddress,
-            constructorArguments: [],
-        });
+        // await hre.run("verify:verify", {
```
Comment (mzxyz) [inline hardhat.config.ts:59]: [Question] why comment out these codes?

**Review by mzxyz — APPROVED** [review]

File: hardhat.config.ts (hunk 1 above)
Comment (Jianzhuo) [reply hardhat.config.ts:59]: it is used for verify contract on polygonscan last time

**Review by nz-neooo — APPROVED** [review]

**Review by mzxyz — APPROVED** [review]

**Pull request merged**